cedar-drt/fuzz/src/schemas.rs

```diff
@@ -0,0 +1,95 @@
+use cedar_policy_validator::{ActionType, ApplySpec, NamespaceDefinition, SchemaFragment};
+
+/// Check if two schema fragments are equivalent
+pub fn semantic_equality_check(lhs: SchemaFragment, rhs: SchemaFragment) -> Result<(), String> {
```

Comment:
Nit: wouldn't the semantic equality be defined as validation results are the same?

Reply:
Yeah, "semantic equality" might not be the right term here. It's more like "equality modulo action groups". Can you add a comment explaining so we don't get confused if we have to edit this in the future?